tabipy.py

```diff
@@ -42,7 +41,15 @@ def __init__(self, value, header=False, bg_colour=None, text_colour=None,
 
     def _latex_escape_func(self, match): 
         """Replace regex match with latex equivalent"""
-        return self._latex_escape_table[match.group()]
+        return self._latex_escape_table[match.group()]        
+
+    def __repr__(self):
+        text = "value = {},\n".format(self.value)
+        text += "header = {},\n".format(self.header)
+        text += "bg_colour = {},\n".format(self.bg_colour)
+        text += "text_colour = {},\n".format(self.text_colour)
+        text += "col_span = {},\n".format(self.col_span)
```

Comment:
I've actually been thinking about making a library to add nice repr methods to classes. Maybe this will prompt me to do it.